css/src/components/hero.scss

```diff
@@ -0,0 +1,153 @@
+$hero-padding-xs: $layout-4 !default;
+$hero-padding-sm: $layout-5 !default;
+$hero-padding-default: $layout-6 !default;
+$hero-padding-lg: $layout-7 !default;
+
+$hero-min-height-xs: 200px !default;
+$hero-min-height-sm: 300px !default;
+$hero-min-height-default: 400px !default;
+$hero-min-height-lg: 500px !default;
+
+$hero-content-width: 45% !default;
+$hero-content-fluid-width: 45% !default;
+$hero-image-width: calc((100% - 2 * $layout-gap) * 0.55) !default;
+$hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !default;
+
+.hero {
+	@extend %layout-gap;
+
+	display: flex;
+	position: relative;
+	flex-direction: column;
+	align-items: stretch;
+	justify-content: space-between;
+
+	.hero-content {
+		z-index: $zindex-active;
+		padding-block: $hero-padding-sm;
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-end: $hero-padding-sm;
+		}
+	}
+
+	&:not(.hero-image) {
```

Comment:
_Reviewers note_: I've realized it might be very useful to fix the content width to the max reading width as it initially was if there is no image on the right side. Any thoughts?

Reply:
I think that's fine.

Reply:
I think we should likely revert this change. I would prefer to keep the widths shorter, and retain consistency with the view with the image included.

Reply:
Can always make it later, after discussion with design team.